tests/test.ts

```diff
@@ -191,14 +197,13 @@ Deno.test({
         }
 
         {
-            const pri = PrivateKey.Generate();
-            const pub = pri.toPublicKey().hex;
-            // create the channel
-            const ChannelCreation_event = await sign_event_v2(pri, {
-                pubkey: pub,
+            const ctx = InMemoryAccountContext.Generate();
```

Comment:
Can a private key also implement the signer v2 interface? Just a quesiton for thoughts

Reply:
Why do we need to do this?

Reply:
We don't have a use case because we rarely direclty use `PrivateKey`